src/api/index.ts

```diff
@@ -1,27 +1,36 @@
 import {AptosClient, Types} from "aptos";
+import {isHex} from "../pages/utils";
 import {withResponseError} from "./client";
 
 export async function getTransactions(
   requestParameters: {start?: number; limit?: number},
   nodeUrl: string,
-): Promise<Types.OnChainTransaction[]> {
+): Promise<Types.Transaction[]> {
   const client = new AptosClient(nodeUrl);
+  const {start, limit} = requestParameters;
+  let bigStart;
+  if (start && 0 <= start && start <= Number.MAX_SAFE_INTEGER) {
```

Comment:
I think this should be an assertion; if a number is passed in that's out of range, it's probably a programmer error and thus should blow up to notify the programmer (rather than silently omitting the param).

Reply:
I think it goes out of scope, I will remove the `0 <= start && start <= Number.MAX_SAFE_INTEGER` for now and add a task to handle it later.